src/components/widget/RawWidget.js

```diff
@@ -322,7 +322,7 @@ class RawWidget extends Component {
                             type="text"
                             ref={c => this.rawWidget = c}
                             className="input-field js-input-field"
-                            value={widgetData[0].value}
+                            defaultValue={widgetData[0].value}
```

Comment:
IMHO, it isnt fully fixing issue. Here you are only hotfixing it, by disabling connection with store of app - so rest of the functionality in our system is 'suspended' as a useless.

If we want to resign with connecting inputs, we have do a major redesign of widgets and window's data structure.

If we want to stay with that, as i commented in issue, we have to fix somehow race with parsing value while typing into input.